Source/UI/NewUI/BillingForm/Default Implementation/BillingForm/Header/Cancel/DefaultCancelButtonFormStyle.swift

```diff
@@ -13,8 +13,9 @@ public struct DefaultCancelButtonFormStyle: ElementButtonStyle {
     public var focusBorderColor: UIColor = .clear
     public var errorBorderColor: UIColor = .clear
     public var imageTintColor: UIColor = .clear
-    public var isHidden: Bool = false
-    public var isEnabled: Bool = true
+    public var isHidden = false
+    public var isEnabled = true
+    public var shouldImageFlippedForRightToLeftLayoutDirection = false
```

Comment:
Is there any trail of why this is needed?

Since the image is provided by client, they can decide to flip it when they declare the style. Not sure we should duplicate that, or that the extra property is justified.

@deepesh-vasthimal-cko ?

Reply:
If device language is in English, and App in Arabic. all images button will flipped. They have the control to flip it or not.